src/nvim/ex_docmd.c

```diff
@@ -7489,9 +7489,8 @@ void do_exedit(exarg_T *eap, win_T *old_curwin)
   if ((eap->cmdidx == CMD_new
        || eap->cmdidx == CMD_tabnew
        || eap->cmdidx == CMD_tabedit
-       || eap->cmdidx == CMD_vnew
-       ) && *eap->arg == NUL) {
-    // ":new" or ":tabnew" without argument: edit an new empty buffer
+       || eap->cmdidx == CMD_vnew) && *eap->arg == NUL) {
```

Comment:
unrelated changes here -- is that to keep lint from complaining?

Reply:
Yep